halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/kubernetes/KubernetesAccount.java

```diff
@@ -96,14 +96,17 @@ public void accept(ConfigProblemSetBuilder psBuilder, Validator v) {
   public void makeBootstrappingAccount(ArtifactSourcesConfig artifactSourcesConfig) {
     super.makeBootstrappingAccount(artifactSourcesConfig);
 
+    // We may want this to use the value from the deployment config rather than the hard-code.
+    String location = "spinnaker";
```

Comment:
This _spinnaker_ value is the actual key that is being used in the config yaml so we might not want to change that.